src/viewport/index.tsx

```diff
@@ -540,32 +528,14 @@ class Viewport extends React.Component<ViewportProps, ViewportState> {
         const totalElapsedTime = now - this.startTime;
         if (elapsedTime > timePerFrame) {
             if (simulariumController.isChangingFile) {
-                this.visGeometry.clearForNewTrajectory();
-                const p = this.visGeometry.mapFromJSON(
-                    simulariumController.getFile(),
-                    simulariumController.getGeometryFile(),
-                    simulariumController.getAssetPrefix()
-                );
-
-                p.then(() => {
-                    this.visGeometry.render(totalElapsedTime);
-                    this.lastRenderTime = Date.now();
-                    this.lastRenderedAgentTime = -1;
-                    this.animationRequestID = requestAnimationFrame(
-                        this.animate
-                    );
-                });
-
-                p.catch(() => {
-                    this.visGeometry.render(totalElapsedTime);
-                    this.lastRenderTime = Date.now();
-                    this.lastRenderedAgentTime = -1;
-                    this.animationRequestID = requestAnimationFrame(
-                        this.animate
-                    );
-                });
-
+                this.visGeometry.render(totalElapsedTime);
+                this.lastRenderTime = Date.now();
+                this.lastRenderedAgentTime = -1;
                 simulariumController.markFileChangeAsHandled();
+           
+                this.animationRequestID = requestAnimationFrame(
+                    this.animate
+                );
```

Comment:
Is it ok to call this after line 534? (since it was being called before, it looks like?)

Reply:
yeah, this mark as handled is a little weird here now because before it was waiting for an async event to be resolved, but now it's just being called in line